src/terrain/clients/notifications.clj

```diff
@@ -70,6 +71,17 @@
                                 :team_name team-name
                                 :admin_message message}}))
 
+(defn send-team-add-notification
+  "Sends a notification indicating that the userhas been added to a team."
+  [user team-name]
+  (send-notification {:type "team"
+                      :user (:id user)
+                      :subject (str "Added to team")
```

Comment:
Probably don't need the `(str )` wrapper here, seems a bit confusing to keep it.

Reply:
Heh, oops! I was originally going to assemble a string and decided not to. I'll fix that.